LootLockerServerSDK/Source/LootLockerServerSDK/Private/ServerAPI/LootLockerServerCharacterRequest.cpp

```diff
@@ -0,0 +1,59 @@
+// Fill out your copyright notice in the Description page of Project Settings.
+
+
+#include "ServerAPI/LootLockerServerCharacterRequest.h"
+
+#include "LootLockerServerHttpClient.h"
+
+ULootLockerServerCharacterRequest::ULootLockerServerCharacterRequest()
+{
+}
+
+void ULootLockerServerCharacterRequest::GetPlayerCharacters(int PlayerID, const FLootLockerServerGetPlayerCharacterResponseBP& OnCompletedRequestBP, const FLootLockerServerGetPlayerCharacterResponseDelegate& OnCompletedRequest)
+{
+    ULootLockerServerHttpClient::SendRequest<FLootLockerServerGetPlayerCharactersResponse>(FLootLockerServerEmptyRequest{}, ULootLockerServerEndpoints::GetPlayerCharacters, { PlayerID }, {}, OnCompletedRequestBP, OnCompletedRequest);
+}
+
+void ULootLockerServerCharacterRequest::GetPlayerCharacterInventory(int PlayerID, int CharacterID, const FLootLockerServerGetCharacterInventoryResponseBP& OnCompletedRequestBP, const FLootLockerServerGetCharacterInventoryResponseDelegate& OnCompletedRequest)
+{
+    ULootLockerServerHttpClient::SendRequest<FLootLockerServerGetCharacterInventoryResponse>(FLootLockerServerEmptyRequest{}, ULootLockerServerEndpoints::GetCharacterInventory, { PlayerID, CharacterID }, {}, OnCompletedRequestBP, OnCompletedRequest);
+}
+
+void ULootLockerServerCharacterRequest::GetPaginatedPlayerCharacterInventory(int PlayerID, int CharacterID, int Count, int After, const FLootLockerServerGetCharacterInventoryResponseBP& OnCompletedRequestBP, const FLootLockerServerGetCharacterInventoryResponseDelegate& OnCompletedRequest)
+{
+    TMultiMap<FString, FString> QueryParams;
+    if(Count > 0)
+    {
+        QueryParams.Add("count", FString::FromInt(Count));
+    }
+    if (After > 0)
+    {
+        QueryParams.Add("after", FString::FromInt(After));
+    }
+    ULootLockerServerHttpClient::SendRequest<FLootLockerServerGetCharacterInventoryResponse>(FLootLockerServerEmptyRequest{}, ULootLockerServerEndpoints::GetCharacterInventory, { PlayerID, CharacterID }, QueryParams, OnCompletedRequestBP, OnCompletedRequest);
+}
+
+void ULootLockerServerCharacterRequest::GetPlayerCharacterLoadout(int PlayerID, int CharacterID, const FLootLockerServerGetCharacterLoadoutResponseBP& OnCompletedRequestBP, const FLootLockerServerGetCharacterLoadoutResponseDelegate& OnCompletedRequest)
+{
+    ULootLockerServerHttpClient::SendRequest<FLootLockerServerGetCharacterLoadoutResponse>(FLootLockerServerEmptyRequest{}, ULootLockerServerEndpoints::GetCharacterLoadout, { PlayerID, CharacterID }, {}, OnCompletedRequestBP, OnCompletedRequest);
+}
+
+void ULootLockerServerCharacterRequest::EquipAssetToPlayerCharacterLoadoutByAssetInstanceId(int PlayerID, int CharacterID, int AssetInstanceID, const FLootLockerServerEquipAssetToCharacterLoadoutResponseBP& OnCompletedRequestBP, const FLootLockerServerEquipAssetToCharacterLoadoutResponseDelegate& OnCompletedRequest)
```

Comment:
Writing this just once here, but same as before change all occurences of assetID, IsntanceID etc. to AssetInstanceID.